scripts/emr/terraform/variables.tf.json

```diff
@@ -0,0 +1,33 @@
+{ 
+  "variable": {
+    "access_key": { 
+        "description": "From your `~/.aws/credentials"
+    },
+    "secret_key": {
+        "description": "From your `~/.aws/credentials"
+    },
+    "pem_path": {
```

Comment:
I think that it would be more convenient to specify `key_name` rather than the path to the key on the filesystem.

Reply:
by introducing a `key_name` you want to introduce a `key_path`? Or to hardcode path to aws key?

Reply:
`key_name` can be brought in through a variable.  It would replace `key_path`.  Please see the terraform configuration [here](https://github.com/geodocker/geodocker-jupyter-geopyspark).

Reply:
@jamesmcclain your links in both last comments are broken, can you fix them?

Reply:
https://github.com/geodocker/geodocker-jupyter-geopyspark